src/Faker/Provider/fa_IR/PhoneNumber.php

```diff
@@ -11,6 +11,15 @@ class PhoneNumber extends \Faker\Provider\PhoneNumber
         '021########',
         '026########',
         '031########',
+        '025########',
+        '051########',
+        '031########',
```

Comment:
it's already there... please ordrer them alphabetically.

Reply:
removed duplicate format